BUILD.md

```diff
@@ -39,14 +59,28 @@
     cmake .. -DBUILD_TESTS=1
     ```
 
-4. **Build the Project**
+1. **Build the Project**
```

Comment:
This description is repeated.

Reply:
Fixed!

Reply:
I think we can put it all together in one block:

```shell
cmake -B build
cmake --build build
```

The second command is more portable and guarantees that we're using the build options set by CMake. Normally, it's equivalent to `make`.

Reply:
Done!